xds/src/test/java/io/grpc/xds/XdsClientImplTest.java

```diff
@@ -2654,7 +2664,7 @@ private static VirtualHost buildVirtualHost(List<String> domains, String cluster
   }
 
   private static Cluster buildCluster(String clusterName, @Nullable String edsServiceName,
```

Comment:
consider adding new method like `buildSecureCluster` which takes the NonNull `UpstreamTlsContext`